src/ring-call-channel.c

```diff
@@ -544,6 +553,15 @@ ring_call_channel_class_init(RingCallChannelClass *klass)
       G_PARAM_READABLE | G_PARAM_STATIC_STRINGS));
 
   g_object_class_install_property(
+    object_class, PROP_IMSI,
+    g_param_spec_string("imsi",
+      "IMSI",
+      "Internation Mobile Subscriber Identifer",
```

Comment:
Ok, this value is the 'blurb' here: https://developer.gnome.org/gobject/stable/gobject-GParamSpec.html#g-param-spec-internal
It is intended to be used in a tooltip or similar indicator within a property editor, so irrelevant to operation.  

I will change them all to be correct.